apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -695,6 +717,12 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
 
     true_answer_list = data["correct_answers"][answer_name]
 
+    distractor_feedback = {
```

Comment:
Not 1000% sure, but it does't look like the values in this dict are ever accessed (only the keys). Also uses the inner HTML instead of a UUID for the key.